examples/documentation_examples/examples.go

```diff
@@ -2712,3 +2712,57 @@ func IndexExamples(t *testing.T, db *mongo.Database) {
 		require.NoError(t, err)
 	}
 }
+
+// VersionedAPIExamples contains examples of creating clients with versioned API.
+func VersionedAPIExamples(t *testing.T) {
+	ctx := context.Background()
+
+	{
+		// Start Versioned API Example 1
+
+		serverAPI := options.ServerAPI(options.ServerAPIVersion1)
+		serverAPIClient, err := mongo.Connect(ctx, options.Client().SetServerAPIOptions(serverAPI))
+
+		// End Versioned API Example 1
+		if err != nil {
```

Comment:
You can use require for these checks. We should probably switch this directory over to using assert, but that sounds like a separate ticket

Reply:
Ah good point. Done here and elsewhere. Filed [GODRIVER-2005](https://jira.mongodb.org/browse/GODRIVER-2005).